sdk/confidentialledger/Azure.Security.ConfidentialLedger/api/Azure.Security.ConfidentialLedger.netstandard2.0.cs

```diff
@@ -5,6 +5,7 @@ public partial class ConfidentialLedgerClient
         protected ConfidentialLedgerClient() { }
         public ConfidentialLedgerClient(System.Uri ledgerUri, Azure.Core.TokenCredential credential) { }
         public ConfidentialLedgerClient(System.Uri ledgerUri, Azure.Core.TokenCredential credential, Azure.Security.ConfidentialLedger.ConfidentialLedgerClientOptions options) { }
+        public ConfidentialLedgerClient(System.Uri ledgerUri, System.Security.Cryptography.X509Certificates.X509Certificate2 clientCertificate, Azure.Security.ConfidentialLedger.ConfidentialLedgerClientOptions options = null) { }
```

Comment:
Could they not just use a `ClientCertificateCredential`? We had talked about this in some form a while ago, but I must apologize that I forgot what we decided. IIRC, you were going to modify it in some way, or further abstract it to handle some case. Sorry, details have escaped me.